fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java

```diff
@@ -780,15 +781,106 @@ public List<Comparable> getShowInfo() throws DdlException {
             jobInfo.add(TimeUtils.longToTimeString(loadStartTimestamp));
             // load end time
             jobInfo.add(TimeUtils.longToTimeString(finishTimestamp));
-            // tracking url
-            jobInfo.add(loadingStatus.getTrackingUrl());
+            // tracking sql
+            if (!loadingStatus.getTrackingUrl().equals(EtlStatus.DEFAULT_TRACKING_URL)) {
+                jobInfo.add("select tracking_msg from information_schema.loads where job_id=" + id);
```

Comment:
A single routine load job possibly returns multi error messages. How to use job_id to identify the error message?

Reply:
this code work for BrokerLoad, will not change behavior of routine load.

Reply:
I see. I wonder whether this is a unified method to track the error message of load?